plugins/modules/dedicated_server_display_name.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import (absolute_import, division, print_function)
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: dedicated_server_display_name
+short_description: Modify the server display name in ovh manager
+description:
+    - Modify the server display name in ovh manager, to help you find your server with your own naming
+author: Synthesio SRE Team
+requirements:
+    - ovh >= 0.5.0
+options:
+    service_name:
+        required: true
+        description: The service name
+    display_name:
+        required: true
+        description: The display name to set
+
+'''
+
+EXAMPLES = '''
+synthesio.ovh.display_name
+  service_name: "{{ ovhname }}"
+  display_name: "{{ ansible_hostname }}"
+delegate_to: localhost
+'''
+
+RETURN = ''' # '''
+
+from ansible_collections.synthesio.ovh.plugins.module_utils.ovh import ovh_api_connect, ovh_argument_spec
+
+try:
+    from ovh.exceptions import APIError
+    HAS_OVH = True
+except ImportError:
+    HAS_OVH = False
+
+
+def run_module():
+    module_args = ovh_argument_spec()
+    module_args.update(dict(
+        display_name=dict(required=True),
+        service_name=dict(required=True)
+    ))
+
+    module = AnsibleModule(
+        argument_spec=module_args,
+        supports_check_mode=True
+    )
+    client = ovh_api_connect(module)
+
+    display_name = module.params['display_name']
+    service_name = module.params['service_name']
+
+    if module.check_mode:
```

Comment:
Can we move this check after the `GET /serviceInfos` to have the error "serviceName not found" even in check mode?